Description: Offer Ubuntu specific NTP servers; pool.ntp.org offers geolocation and automatic QA
Author: Xavier Robin
Bug-Ubuntu: https://launchpad.net/bugs/104525

Index: gnome-system-tools-3.0.0/src/time/ntp-servers-list.c
===================================================================
--- gnome-system-tools-3.0.0.orig/src/time/ntp-servers-list.c	2010-12-12 07:16:01.000000000 -0500
+++ gnome-system-tools-3.0.0/src/time/ntp-servers-list.c	2011-12-16 13:23:48.625087868 -0500
@@ -34,33 +34,12 @@
 
 struct NtpServer {
 	const gchar *url;
-	const gchar *location;
 } ntp_servers [] = {
-	{ "time.nrc.ca", "Canada" },
-	{ "ntp1.cmc.ec.gc.ca", "Eastern Canada" },
-	{ "ntp2.cmc.ec.gc.ca", "Eastern Canada" },
-	{ "wuarchive.wustl.edu", "Missouri, USA" },
-	{ "clock.psu.edu", "Pennsylvania, USA" },
-	{ "ntp.tmc.edu", "Texas, USA" },
-	{ "louie.udel.edu", "Delaware, USA" },
-	{ "ntp0.cornell.edu", "New York, USA" },
-	{ "ntp-0.cso.uiuc.edu", "Illinois, USA" },
-	{ "ntp1.cs.wisc.edu", "Wisconsin, USA" },
-	{ "tick.cs.unlv.edu", "Las Vegas, USA" },
-	{ "ntp2a.mcc.ac.uk", "England, Europe" },
-	{ "ntp2b.mcc.ac.uk", "England, Europe" },
-	{ "ntp.maths.tcd.ie", "Ireland, Europe" },
-	{ "ntp.cs.strath.ac.uk", "Scotland, Europe" },
-	{ "ntp.univ-lyon1.fr", "France, Europe" },
-	{ "tick.keso.fi", "Finland, Europe" },
-	{ "fartein.ifi.uio.no", "Norway, Europe" },
-	{ "ntp.adelaide.edu.au", "South Australia" },
-	{ "time.nuri.net", "Korea, Asia" },
-	{ "ntp.cs.mu.oz.au", "Melbourne, Australia" },
-	{ "swisstime.ethz.ch", "Zurich, Switzerland" },
-	{ "stdtime.gov.hk", "Hong Kong, China" },
-	{ "ntp.jst.mfeed.ad.jp", "Japan" },
-	{ "time.service.uit.no", "Norway" },
+	{ "0.ubuntu.pool.ntp.org" },
+	{ "1.ubuntu.pool.ntp.org" },
+	{ "2.ubuntu.pool.ntp.org" },
+	{ "3.ubuntu.pool.ntp.org" },
+	{ "ntp.ubuntu.com" },
 	{ NULL }
 };
 
@@ -72,7 +51,7 @@
 	gint i;
 
 	for (i = 0; ntp_servers[i].url; i++) {
-		str = g_strdup_printf ("%s (%s)", ntp_servers[i].url, ntp_servers[i].location);
+		str = g_strdup_printf ("%s", ntp_servers[i].url);
 
 		gtk_list_store_append (store, &iter);
 		gtk_list_store_set (store, &iter,
